bin/build-hdfs

```diff
@@ -42,11 +45,11 @@ fi
 # Get native libraries
 if [ ! -d $NATIVE ]; then
     echo "Downloading and unpacking native libs"
-    wget https://github.com/cloudera/Impala/archive/cdh5-1.4_5.1.0.zip || exit
-    unzip -q cdh5-1.4_5.1.0.zip
+    wget $IMPALA_URL || exit
+    unzip -q $IMPALA_VER.zip
     mkdir -p $NATIVE
-    cp Impala-cdh5-1.4_5.1.0/thirdparty/hadoop-2.3.0-cdh5.1.0/lib/native/lib* $NATIVE
-    rm -rf cdh5-1* Impala*
+    cp Impala-$IMPALA_VER/thirdparty/$HADOOP_DIR/lib/native/lib* $NATIVE
```

Comment:
Is this guaranteed to be the same as $HADOOP_DIR? Seems like we're tying two dependencies together that may not necessarily need to be.